### PR TITLE
Private Events

### PR DESCRIPTION
**Pull request by BigWizzo**
> Ruby on Rails - Private Events:
- We created this app to allow users to create events and manage user signups. Users can create events and send invitations. A user can attend many events, and many users can attend an event.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+<p>You are logged in: <%= current_user.username %> </p>
+
+<%= link_to "Log Out", session_path(session[:username]), method: :delete %><br>
+
+<% else %>
+
+<%= link_to "Log In", new_session_path %>
+
+<% end %>
```
Comment (teekaytech) [inline app/views/users/index.html.erb:13]: There should be no logic in your views. Kindly use the helper module to encapsulate view-related logic. Please check through all your view files and make the necessary changes.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this project. I like the simplicity of your design.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] The user's show page displays events the user has created and events that the user is attending, **separated in past and upcoming** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
- At the moment, the displayed events are not separated into past and upcoming events in the user show page (My Events).

- [x] You have a commented line of code in your `app/controllers/events_controller.rb` script. Kindly remove it.

- [x] There should be absolutely NO logic in views. Views are only for displaying information that became available to it from its corresponding controller not generating it.
      **Resources**
- [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
- [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
- [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

- [x] Check my inline comments below for details of other required changes.


**[ OPTIONAL CHANGES ]**

- [ ] At the moment, a user can attend an event as many times as possible. Kindly ensure that a user cannot attend a single event multiple times. [This stack overflow thread](https://stackoverflow.com/questions/16735454/correct-way-of-prevent-duplicate-records-in-rails) might help.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] please delete `app/views/users/test.erb` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
### Hey there :wave: 
#### Well done in making the changes requested of you.
#### The project is now approved and you can go ahead and merge this branch and good luck with the next project.
slack(@ebeagusamuel)

**Pull request merged**